src/Main/DetectiveGameMain.java

```diff
@@ -0,0 +1,132 @@
+package Main;
+
+import Commands.Command;
+import Commands.CommandFactory;
+import Core.*;
+import Extractors.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+//for New Branch
+
+public class DetectiveGameMain {
+    public static void main(String[] args) {
+        List<CaseFile> cases = loadCases();
+        displayCaseMenu(cases);
+        CaseFile selectedCase = selectCase(cases);
+        startGame(selectedCase);
+    }
+
+    private static List<CaseFile> loadCases() {
+        List<CaseFile> cases = new ArrayList<>();
+        ObjectMapper mapper = new ObjectMapper();
+        File folder = new File("cases");
```

Comment:
Don't hardcode this, pass from environment variable or program arguments like in the labs